system/react/src/components/Radio.ts

```diff
@@ -0,0 +1,73 @@
+import styled from '@emotion/styled';
+
+export const classlessSelector = 'input[type="radio"]';
+export const classySelector = '.radio';
+
+export const Radio = styled.input<{ type?: 'radio' }>`
```

Comment:
This should be required.
```suggestion
export const Radio = styled.input<{ type: 'radio' }>`
```

Reply:
Why, it's set from default props, so always would be radio?

Reply:
Missed that, I completely forgot defaultProps was a thing 🤣